src/Controller/DebugPagesController.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\PageBundle\Controller;
+
+use Doctrine\ORM\EntityRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Zicht\Bundle\PageBundle\Model\ContentItemInterface;
+use Zicht\Bundle\PageBundle\Model\PageInterface;
+
+class DebugPagesController extends AbstractController
+{
+    /**
+     * @Route("debug/pages")
```

Comment:
This route should be moved to a separate YAML file so you can include it in dev and test environments, but not in prod.

Reply:
Je kunt ook via `condition="%kernel.debug%"` dit bijv. afdwingen